src/main/java/com/google/sps/servlets/CreateRoomServlet.java

```diff
@@ -0,0 +1,99 @@
+package com.google.sps.servlets;
+
+import com.google.sps.data.*;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.Key;
+import com.google.gson.Gson;
+import java.util.Queue;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import com.google.gson.JsonArray;
+import java.util.Arrays;
+import com.google.appengine.api.datastore.EmbeddedEntity;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.DatastoreFailureException;
+import com.google.appengine.api.datastore.QueryResultList;
+
+@WebServlet("/create-room")
+public final class CreateRoomServlet extends HttpServlet {
+    public static final DatastoreService DATASTORE = DatastoreServiceFactory.getDatastoreService();
+
+    public void doPost(HttpServletRequest req, HttpServletResponse res) throws IOException {
+
+        String emails = req.getParameter("Invitees").replace(" ", "");
+        List<Member> members = new ArrayList<Member>();
+        for(String m : Arrays.asList(emails.split(","))){
```

Comment:
no need to wrap emails.split(",") with a Arrays.asList(...). 

However, if you did Arrays.asList(emails.split()).stream().map(Member::createNewMember).collect(Collectors.toList()); 

Then I think it'd make sense. I'll leave you to decide.